src/dxvk/dxvk_graphics.h

```diff
@@ -617,6 +621,13 @@ namespace dxvk {
 
     alignas(CACHE_LINE_SIZE)
     dxvk::mutex                                   m_mutex;
+    alignas(CACHE_LINE_SIZE)
+    dxvk::mutex                                   m_asyncMutex;
+
+    bool                                          m_async = false;
```

Comment:
And another small change from Britt Yazel DXVK GPLAsync patch.
```suggestion
   std::atomic<bool>                             m_async{false};
```